desc/plotting.py

```diff
@@ -1282,6 +1282,132 @@ def plot_surfaces(eq, rho=8, theta=8, zeta=None, ax=None, **kwargs):
     return fig, ax
 
 
+def plot_boundary(eq, zeta=None, axis=True, ax=None, **kwargs):
+    """Plot stellarator boundary at multiple toroidal coordinates.
+
+    Parameters
+    ----------
+    eq : Equilibrium
```

Comment:
This might be a good place to allow passing in a `Surface` object directly. IE, make the plot routine expect a surface, or use `eq.surface`?